Source/Core/sampleTerrain.js

```diff
@@ -1,11 +1,17 @@
 define([
         '../ThirdParty/when',
-        './Check'
+        './defined',
+        './Check',
+        './LRUCache'
     ], function(
         when,
-        Check) {
+        defined,
+        Check,
+        LRUCache) {
     'use strict';
 
+    var cache = new LRUCache(100, 60000);
```

Comment:
What's a reasonable capacity here?  Since entries in the cache expire and get removed after 1 minute, should I maybe make the capacity larger?